bin/templates/cordova/lib/PackageJsonParser.js

```diff
@@ -29,13 +30,28 @@ class PackageJsonParser {
         };
     }
 
-    configure (config) {
+    configure (config, projectPackageJson) {
         if (config) {
             this.package.name = config.packageName() || 'io.cordova.hellocordova';
             this.package.displayName = config.name() || 'HelloCordova';
             this.package.version = config.version() || '1.0.0';
             this.package.description = config.description() || 'A sample Apache Cordova application that responds to the deviceready event.';
 
+            const cordovaDependencies = Object.keys(projectPackageJson.dependencies)
+                .filter((npmPackage) => /^cordova(?!-plugin)-/.test(npmPackage));
+
+            // If Cordova dependencies are detected in "dependencies" of "package.json" warn for potential app package bloating
+            if (cordovaDependencies.length) {
+                events.emit('warn', `The following Cordova package(s) were detected as "dependencies" in the projects "package.json" file.
+\t- ${cordovaDependencies.join('\n\t- ')}
+
+It is recommended that all Cordova packages are defined as "devDependencies" and is safe to be moved manually.
+Note: Defined package as a dependency will be bundled with the application and can increase the built application's size greatly.
```

Comment:
```suggestion
Motivation: Packages defined as a dependency will be bundled with the application and can increase the built application's size.
```

Reply:
For some reason, I am not really all for replacing the word `Note` with `Motivation`... 

Some reason `Motivation` does not feel like the right word to use in a warning statement on the CLI output. But the other changes are great.

Could drop `Note:` all together if thats better?

Reply:
done